### PR TITLE
refactor: change nft capitalization from capitalized to camelcase

### PR DESCRIPTION
**Pull request by 0x4007**
We have some inconsistencies on how we refer to "NFT" within variable names as I was working on the UI. We should proceed by using camelCase (Nft) instead of uppercase (NFT)

### CODE REVIEW

File: packages/contracts/test/manager/DiamondTestSetup.sol [hunk 1]
```diff
@@ -169,7 +169,7 @@ abstract contract DiamondSetup is DiamondTestHelper {
             "ManagerFacet"
         ];
 
-        // diamod arguments
+        // diamond arguments
```
Comment (0x4007) [inline packages/contracts/test/manager/DiamondTestSetup.sol:172]: I thought this was fixed in another PR so this is unexpected that I had to manually change again.

File: packages/contracts/test/manager/DiamondTestSetup.sol [hunk 2]
```diff
@@ -215,7 +215,7 @@ abstract contract DiamondSetup is DiamondTestHelper {
         diamond = new Diamond(_args, diamondCut);
         vm.stopPrank();
 
-        // initialise interfaces
+        // initialize interfaces
```
Comment (0x4007) [inline packages/contracts/test/manager/DiamondTestSetup.sol:218]: Same here.

**Comment by 0x4007** [comment]
### Notice

CI isn't working because I changed the filenames (NFT -> Nft). Perhaps we could have another CI test that runs on the current PR vs off of development?

### UbiquiStick Not Building

I think I screwed up the build by updating the `forge` dependencies with more updated OpenZepplin contracts (UbiquiStick doesn't compile anymore)

```log
Compiler run failed
error[7792]: TypeError: Function has override specified but does not override anything.
   --> src/ubiquistick/UbiquiStick.sol:124:16:
    |
124 |     ) internal override(ERC721, ERC721Enumerable) {
    |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

error[2353]: TypeError: Invalid contracts specified in override list: "ERC721" and "ERC721Enumerable".
   --> src/ubiquistick/UbiquiStick.sol:124:16:
    |
124 |     ) internal override(ERC721, ERC721Enumerable) {
    |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Note: This contract:
  --> lib/openzeppelin-contracts/contracts/token/ERC721/ERC721.sol:19:1:
   |
19 | contract ERC721 is Context, ERC165, IERC721, IERC721Metadata {
   | ^ (Relevant source part starts here and spans across multiple lines).
Note: This contract:
  --> lib/openzeppelin-contracts/contracts/token/ERC721/extensions/ERC721Enumerable.sol:14:1:
   |
14 | abstract contract ERC721Enumerable is ERC721, IERC721Enumerable {
   | ^ (Relevant source part starts here and spans across multiple lines).

error[7792]: TypeError: Function has override specified but does not override anything.
   --> src/ubiquistick/UbiquiStick.sol:133:16:
    |
133 |     ) internal override(ERC721, ERC721Enumerable) {
    |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

error[2353]: TypeError: Invalid contracts specified in override list: "ERC721" and "ERC721Enumerable".
   --> src/ubiquistick/UbiquiStick.sol:133:16:
    |
133 |     ) internal override(ERC721, ERC721Enumerable) {
    |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Note: This contract:
  --> lib/openzeppelin-contracts/contracts/token/ERC721/ERC721.sol:19:1:
   |
19 | contract ERC721 is Context, ERC165, IERC721, IERC721Metadata {
   | ^ (Relevant source part starts here and spans across multiple lines).
Note: This contract:
  --> lib/openzeppelin-contracts/contracts/token/ERC721/extensions/ERC721Enumerable.sol:14:1:
   |
14 | abstract contract ERC721Enumerable is ERC721, IERC721Enumerable {
   | ^ (Relevant source part starts here and spans across multiple lines).

error[6480]: TypeError: Derived contract must override function "_beforeTokenTransfer". Two or more base classes define function with same name and parameter types.
  --> src/ubiquistick/UbiquiStick.sol:24:1:
   |
24 | contract UbiquiStick is
   | ^ (Relevant source part starts here and spans across multiple lines).
Note: Definition in "ERC721":
   --> lib/openzeppelin-contracts/contracts/token/ERC721/ERC721.sol:437:5:
    |
437 |     function _beforeTokenTransfer(
    |     ^ (Relevant source part starts here and spans across multiple lines).
Note: Definition in "ERC721Enumerable":
  --> lib/openzeppelin-contracts/contracts/token/ERC721/extensions/ERC721Enumerable.sol:60:5:
   |
60 |     function _beforeTokenTransfer(
   |     ^ (Relevant source part starts here and spans across multiple lines).

error[6160]: TypeError: Wrong argument count for function call: 3 arguments given but expected 4.
   --> src/ubiquistick/UbiquiStick.sol:125:9:
    |
125 |         super._beforeTokenTransfer(from, to, tokenId);
    |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

**Comment by Steveantor** [comment]
> We have some inconsistencies on how we refer to "NFT" within variable names as I was working on the UI. We should proceed by using camelCase (Nft) instead of uppercase (NFT)

As far as I've seen, most projects tend to use NFT instead of Nft, since
that's the standard for abbreviations.

**Comment by pavlovcik** [comment]
I'm still looking for help on compiling the UbiquiStick contract. I'm not sure what the solution is here.

**Comment by 0x4007** [comment]
I renamed the files and now CI is breaking. I think I'll need to just merge this in and see on `development` but it builds locally.

Update: I just realized that git doesn't track same file names if only the capitalization is changed. Working on that now.

**Comment by pavlovcik** [comment]
It builds locally not sure what to do here.

**Pull request closed without merging**